mu/sims/pedal_board.py

```diff
@@ -1,7 +1,22 @@
 from mu.harness.board_sim import BoardSim
-from mu.sims.sub_sims.ads1015 import Ads1015
+from mu.harness.sim_io import SimIo
+from mu.sims.sub_sims.ads1015 import Ads1015, ADS1015_KEY
 
 
 class PedalBoard(BoardSim):
     def __init__(self, pm, proj_name='pedal_board'):
         super().__init__(pm, proj_name, sub_sim_classes=[Ads1015])
+        self.pm.new_io(SimIo('throttle', self.get_throttle, self.set_throttle))
+        self.pm.new_io(SimIo('brake', self.get_brake, self.set_brake))
```

Comment:
These need to get cleaned up somehow when a sim gets stopped, otherwise this will error out:
```
musrv -bg
muctl start PedalBoard
muctl stop PedalBoard
muctl start PedalBoard # gives 'SimIo name taken'
```
and after that `muctl get` refers to the wrong PedalBoard instance (and `muctl set` is timing out for some reason).